.wordlist.txt

```diff
@@ -64,6 +64,7 @@ DataLoader
 DataParallel
 DeepSpeed
 Dependabot
+Distro
```

Comment:
@LisaDelaney I am off the opinion we should stick with the formal Linux Distribution, versus distro

Reply:
We will have to update this image then as well in the install docs
![image](https://github.com/RadeonOpenCompute/rocm-docs-core/assets/22262939/2b6f7ddf-9032-421b-8451-f4aa94177293)

Reply:
Agreed, distro is slang/very informal.